pkg/dockerfile/fast_generator.go

```diff
@@ -0,0 +1,273 @@
+package dockerfile
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/replicate/cog/pkg/config"
+	"github.com/replicate/cog/pkg/weights"
+)
+
+const FUSE_RPC_WEIGHTS_PATH = "/srv/r8/fuse-rpc/weights"
+const MONOBASE_CACHE_PATH = "/var/cache/monobase"
+const APT_CACHE_MOUNT = "--mount=type=cache,target=/var/cache/apt,id=apt-cache"
+const PIP_CACHE_MOUNT = "--mount=type=cache,target=/root/.cache,id=pip-cache"
```

Comment:
This should be `/srv/r8/monobase/uv/cache` as defined here:
https://github.com/replicate/monobase/blob/main/Dockerfile#L28

Reply:
Resolved in [0e019cb](https://github.com/replicate/cog/pull/2108/commits/0e019cb77300143a81a59328184501725c7eabe5)